metaflow/plugins/argo/argo_workflows.py

```diff
@@ -3754,8 +3806,7 @@ class ArgoWorkflowTrigger(object):
     # https://github.com/argoproj/argo-events/blob/master/api/sensor.md#argoproj.io/v1alpha1.ArgoWorkflowTrigger
 
     def __init__(self):
-        tree = lambda: defaultdict(tree)
-        self.payload = tree()
+        self.payload = _tree()
```

Comment:
this change seems unrelated to the PR. can you consider pulling this out as a different PR or skipping it?

Reply:
Yeah, let me revert